src/main/java/com/google/error_analyzer/data/StoreLogs.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.error_analyzer.data;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class StoreLogs {
+
+    private static final Logger LOG = LogManager.getLogger(StoreLogs.class);
+
+    public String convertToJsonString(String logText, String logLineNumber) {
+        logText=logText.replaceAll("[^a-zA-Z0-9:\"]", " ");
```

Comment:
make constant

Reply:
Also, keep the variable name descriptive to explain regex.
Or add a comment explaining the updation.

Reply:
Similarly below.